disktools.py

```diff
@@ -1366,6 +1366,23 @@ def getMdNodes():
         pass
     return nodes
 
+def getMdDevicesUsing(physical_disks):
+    nodes = []
+    disks = list(map(os.path.basename, physical_disks))
+    try:
+        fh = open('/proc/mdstat')
```

Comment:
Use `with open(...` instead to ensure the file is closed if an exception occurs.